engine/lib/phx/src/math/matrix.rs

```diff
@@ -697,15 +699,11 @@ pub unsafe extern "C" fn Matrix_ToQuat(this: &Matrix, q: &mut Quat) {
 
 #[no_mangle]
 pub extern "C" fn Matrix_Print(this: &Matrix) {
-    let mut i: i32 = 0;
-    while i < 4 {
```

Comment:
Yeah, sorry, there are a few of these left over from when C2Rust generated all the Rust code. It basically generated these while loops instead of for loops, i got most of them but not all.